content/references/rippled-api/api-conventions/serialization.md

```diff
@@ -0,0 +1,285 @@
+# Serialization Format
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/STObject.cpp#L696-L718 "Source")
+
+This page describes the XRP Ledger's canonical binary format for transactions and other data. This binary format is necessary to create and verify digital signatures of those transactions' contents, and is also used in other places. The [rippled APIs](rippled-api.html) typically use JSON to communicate with client applications. However, JSON is unsuitable as a format for serializing transactions for being digitally signed, because JSON can represent the same data in many different but equivalent ways.
+
+The process of serializing a transaction from JSON or any other representation into their canonical binary format can be summarized with these steps:
+
+1. Make sure all required fields are provided, including any required but "auto-fillable" fields.
```

Comment:
Would it make sense to link auto-fillable to https://developers.ripple.com/transaction-common-fields.html#auto-fillable-fields? Relevant?

Reply:
Added the link